src/classes/vscommands/i18ncommand/ExportToI18NCommand.ts

```diff
@@ -220,7 +220,14 @@ export class ExportToI18NCommand extends ParserBearer {
 	private async _insertIntoI18NFile(stringToInsert: string) {
 		const [manifest] = this._parser.fileReader.getAllManifests();
 		const manifestFsPath = manifest?.fsPath;
-		const i18nRelativePath = manifest?.content["sap.app"]?.i18n;
+		let i18nRelativePath = manifest?.content["sap.app"]?.i18n;
+		if(typeof i18nRelativePath === "object"){

```

Comment:
Please install prettier and apply it, the project already has .prettierrc for it.